agent/cita_exporter/Dockerfile

```diff
@@ -1,16 +1,25 @@
-FROM ubuntu:18.04
+FROM rust:1.48 AS builder
```

Comment:
should add a comment for what is this sage used for, for example: `# build cita-cli bin`